hkmc2/shared/src/main/scala/hkmc2/codegen/Lifter.scala

```diff
@@ -298,6 +298,7 @@ class Lifter(handlerPaths: Opt[HandlerPaths])(using State, Raise):
       sortedVars.iterator.foldLeft[Block](End()):
         case (acc, (_, _, vd)) => Define(vd, acc),
       N,
+      N, // TODO: capture class is likely bufferable
```

Comment:
Even if it is, how would that help? We can't allocate these on buffers, as there is no way to automaticaly manage their lifetime. Capture class instances can leave the current scope and live indefinitely.